var/spack/repos/builtin/packages/py-continuum/package.py

```diff
@@ -16,6 +16,8 @@ class PyContinuum(PythonPackage):
 
     version("1.2.7", sha256="baadcdbe2f5b3c05254307c73434a012f2a3d663e1be9469a03d4b82559e98e1")
 
+    patch("move_prospector_dev_requires.patch")
```

Comment:
Can you add a comment with a link to the PR? Also, probably want to add:
```suggestion
    patch("move_prospector_dev_requires.patch", when="@:1.2.7")
```
I'm guessing this won't be needed for the next release.